manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "deletedvideos",
+  "version": "1.0",
+  "manifest_version": 2,
+  "content_scripts": [
+    {
+      "matches": ["*://*.youtube.com/playlist?list=*"],
+      "js": ["content.js"]
+    }
+  ],
+  "browser_action": {
```

```suggestion
  "page_action": { 
```
Change to page_action, https://developer.chrome.com/extensions/pageAction